src/webnn_native/Graph.cpp

```diff
@@ -138,27 +137,26 @@ namespace webnn_native {
         return CompileImpl();
     }
 
-    WNNComputeGraphStatus GraphBase::Compute(NamedInputsBase* inputs, NamedOutputsBase* outputs) {
+    void GraphBase::Compute(NamedInputsBase* inputs, NamedOutputsBase* outputs) {
         if (inputs == nullptr || outputs == nullptr) {
-            return WNNComputeGraphStatus_Error;
+            GetContext()->ConsumedError(DAWN_VALIDATION_ERROR("input or output is nullptr."));
```

Comment:
I think we can remove L.142 because it will create error object if there is an error  for the object, so the pointer of NamedInputsBase and NamedOutputsBase aren't nullptr

Reply:
ok